lib/cocoapods-fix-react-native/versions/0_56_0-rc_5-post.rb

```diff
@@ -0,0 +1,88 @@
+require 'cocoapods'
+require 'cocoapods-fix-react-native/helpers/root_helper'
+
+# Obtain the React Native root directory
+$root = get_root
+
+# Detect CocoaPods + Frameworks
+$has_frameworks = File.exists?'Pods/Target Support Files/React/React-umbrella.h'
+
+# TODO: move to be both file in pods and file in node_mods?
+def patch_pod_file(path, old_code, new_code)
+  file = File.join($root, path)
+  unless File.exist?(file)
+    Pod::UI.warn "#{file} does not exist so was not patched.."
+    return
+  end
+  code = File.read(file)
+  if code.include?(old_code)
+    Pod::UI.message "Patching #{file}", '- '
+    FileUtils.chmod('+w', file)
+    File.write(file, code.sub(old_code, new_code))
+  end
+end
+
+def fix_unused_yoga_headers
```

Comment:
@orta Looks like https://github.com/facebook/react-native/commit/cf036dbc7af16a8453c115372694dc51e8086fcf will make this patch unnecessary, right?

Reply:
Yes, I think so 👍

Reply:
Thanks! I'll remove this as well.